website/docs/concepts/event-bridge/integrations.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 5
+---
+
+Support (See [issues](https://github.com/functionless/functionless/issues?q=is%3Aissue+is%3Aopen+label%3Aevent-bridge) for progress):
+
+- Lambda
+- Step Functions
+- Event Bus (Bus to Bus)
```

Comment:
this is new compared to the previous list

maybe in the above page it should be a link to here

Reply:
Two different relationships, integration in and integration out. Does need to be better organized.